pages/dkp/konvoy/1.8/release-notes/1.8.5/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes Konvoy 1.8.5
+title: Release Notes Konvoy 1.8.5
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 1.8.5
+beta: false
+enterprise: false
+---
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Konvoy&reg; version 1.8.5 was released on February 17th, 2022**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.18.x  |
+| **Maximum**  | 1.20.x  |
+| **Default**  | 1.20.15  |
+
+### New features and capabilities
+
+#### CentOS 8 EOL
+
+- CentOS 8 has been removed from build tooling and testing as it has reached end of life (EOL). CentOS 8 have been replaced with Alma Linux 8 when building RHEL 8.x images.
```

Comment:
The 'Alma Linux 8' stuff only applies to Konvoy/DKP 2.x.  We're not doing anything about this in Konvoy 1.x, you just can't use CentOS8 anymore.

```suggestion
- CentOS 8 has reached end of life (EOL), and thus is no longer supported by Konvoy.
```